lib/js/atoms/Checkbox/Checkbox.vue

```diff
@@ -0,0 +1,212 @@
+			'-xsmall': size === CHECKBOX_SIZES.X_SMALL,
+		}"
+	>
+		<v-layout>
+			<v-checkbox
+				:id="id"
+				:disabled="disabled"
+				:error="isErrorComputed"
+				:error-message="errorMessage"
+				:label="label"
+				:messages="errorMessage || helpMessage"
+				:readonly="readonly"
+				:value="value"
+				@click="$emit('click', $event)"
+				@change="$emit('change', $event)"
+				@input="$emit('input', $event)"
+			>
+				<template #label>
+					<div>
+						<span class="a-checkbox__label">
+							{{ label }}
+						</span>
+						<span v-if="labelAddition" class="a-checkbox__labelAddition">
+							{{ labelAddition }}
+						</span>
+					</div>
+				</template>
+
+				<template #message>
+					<div class="a-checkbox__helpMessage">
+						<icon
+							v-if="isErrorComputed"
+							class="a-checkbox__labelErrorIcon"
+							:icon="ICONS.FA_EXCLAMATION_TRIANGLE"
+							:size="ICON_SIZES.XX_SMALL"
+						></icon>
+						<span>
+							{{ errorMessage || helpMessage }}
+						</span>
+					</div>
+				</template>
+			</v-checkbox>
+		</v-layout>
+	</div>
+</template>
+
+<style lang="scss">
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+
+$checkbox-xs-size: 12px;
+$checkbox-s-size: 16px;
+$checkbox-m-size: 20px;
+
+.a-checkbox {
+	&__label {
+		@include headlineS();
+
+		color: $color-rhino-gray;
+	}
+
+	&__labelAddition {
+		color: $color-primary;
+		font-style: italic;
+	}
+
+	&.-error {
+		.a-checkbox__label,
+		.a-checkbox__labelAddition,
+		.a-checkbox__helpMessage {
+			color: $color-danger;
+		}
+	}
+
+	&.-disabled {
+		.a-checkbox__label,
+		.a-checkbox__labelAddition {
+			color: rgba($color-total-black, 0.38);
+		}
+	}
+
+	&.-xsmall {
+		.v-icon {
+			font-size: $checkbox-xs-size;
+		}
+
+		.v-input--selection-controls__input {
+			margin-right: 0;
+		}
+
+		.a-checkbox__label,
+		.a-checkbox__labelAddition {
+			@include headlineXXS();
+		}
+
+		.v-input--selection-controls__input:hover .v-input--selection-controls__ripple::before,
+		.v-input--selection-controls__ripple > .v-ripple__container {
+			transform: scale(0.8);
+		}
+	}
+
+	&.-small {
+		.v-icon {
+			font-size: $checkbox-s-size;
+		}
+
+		.v-input--selection-controls__input {
+			margin-right: $space-xxxxs;
+		}
+
+		.a-checkbox__label,
+		.a-checkbox__labelAddition {
+			@include headlineXS();
+		}
+
+		.v-input--selection-controls__input:hover .v-input--selection-controls__ripple::before,
+		.v-input--selection-controls__ripple > .v-ripple__container {
+			transform: scale(1);
+		}
+	}
+
+	&.-medium {
+		.v-icon {
+			font-size: $checkbox-m-size;
+		}
+
+		.a-checkbox .v-input--selection-controls__input {
+			margin-right: $space-xxs;
+		}
+	}
+}
+</style>
+
+<script lang="ts">
+import { VCheckbox, VLayout } from 'vuetify/lib';
+
+import { CHECKBOX_SIZES } from './Checkbox.consts';
+import Icon, { ICON_SIZES, ICONS } from '../Icon';
+
+export default {
+	name: 'Checkbox',
```

Comment:
https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential

Maybe we should plan to migrate all components names to include `Ds` or `B` prefix?